src/lib/components/quiz/quiz.svelte

```diff
@@ -0,0 +1,233 @@
+<script lang="ts">
+  import type { QuizData } from '$lib/constants/quiz';
+  import LeftArrow from '../icons/leftArrow.svelte';
+  import RightArrow from '../icons/rightArrow.svelte';
+  import MoreInfo from './moreInfo.svelte';
+
+  export let data: QuizData;
+
+  $: index = 0;
+  let response: string[] = [];
+  let noMoreQuestions: boolean = false;
+  let answeredEveryQuestion: boolean = false;
+  let showResults: boolean = false;
+  let showMoreInfo: boolean = false;
+  let showTeam: string;
+  let tallyChoices = {
+    aiChoice: 0,
+    algoChoice: 0,
+    designChoice: 0,
+    devChoice: 0,
+  };
```

Comment:
We might be able to use a complicated reactive statement to calculate the quiz result.
```suggestion
  $: talliedResponses = responses.reduce((tallies, match, i) => {
    if (tallies[match]) tallies[match]++;
    else tallies[match] = 1;
    return tallies;
  }, {});
```

Reply:
Done, removed the i parameter since Svelte was yelling at me that it was not being used when I tried npm run all.

Reply:
Good catch; not sure how that got in there 😅